playwright-tests/tests/connectWallet.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { test } from '../services/fixture.service';
+import { Tags } from '../test-data/tags.data';
+import { ReefKnotPage } from '../pages/reefKnot.page';
+import { expect } from '@playwright/test';
+import { REEF_KNOT_CONFIG } from '../config';
+import { formatEther } from 'viem';
+import { toCut } from '../services/helpers/helpers';
+import { qase } from 'playwright-qase-reporter';
+
+test.describe.serial(
+  'ReefKnot. Wallet connection',
+  { tag: [Tags.connectedWallet] },
+  async () => {
+    let reefKnotPage: ReefKnotPage;
+
+    test.beforeAll(async ({ reefKnotService }) => {
+      reefKnotPage = new ReefKnotPage(reefKnotService.page);
+      await reefKnotPage.goto();
+      await reefKnotPage.allowUseCookies();
+    });
+
+    test.afterAll(async ({ reefKnotService }) => {
+      await reefKnotService.disconnectWalletForce();
+    });
+
+    test(qase(434, 'Connect wallet'), async ({ reefKnotService }) => {
+      await test.step('Check the stand appearance before wallet connection', async () => {
+        await expect(
+          reefKnotPage.statsBlock.mainComponent,
+          'Expected the statistic block is not displayed before wallet connection',
+        ).not.toBeVisible();
+        await expect(
+          reefKnotPage.header.accountButton,
+          'Expected the account button is not displayed before wallet connection',
+        ).not.toBeVisible();
+      });
+      await reefKnotService.connectWallet();
+      await test.step('Check the stand appearance after wallet connection', async () => {
+        await expect(
+          reefKnotPage.statsBlock.mainComponent,
+          'Expected the statistic block is displayed after wallet connection',
+        ).toBeVisible();
+        await expect(
+          reefKnotPage.header.accountButton,
+          'Expected the account button is displayed after wallet connection',
+        ).toBeVisible();
+      });
+    });
+
+    test.describe('Check statistic block', async () => {
```

Comment:
wdyt about moving this describe block to another file and removing serial?
Also, I believe that in connection tests, it's sufficient to verify that the balance is displayed, rather than verifying the balance's correctness itself.

Reply:
done